addons/binding/org.openhab.binding.knx/src/main/java/org/openhab/binding/knx/internal/dpt/KNXCoreTypeMapper.java

```diff
@@ -116,6 +131,9 @@ public KNXCoreTypeMapper() {
 
         dptTypeMap = new HashMap<String, Class<? extends Type>>();
         dptMainTypeMap = new HashMap<Integer, Class<? extends Type>>();
+        defaultDptMap = new HashMap<Class<? extends Type>, String>();
+        defaulItemTypeMap = new HashMap<Class<? extends Type>, Class<? extends GenericItem>>();
+        itemTypeStringMap = new HashMap<Class<? extends GenericItem>, String>();
```

Comment:
Can you use the [diamond operator](https://www.javaworld.com/article/2074080/core-java/jdk-7--the-diamond-operator.html) here?